spaceship/spec/tunes/app_submission_spec.rb

```diff
@@ -58,5 +58,19 @@
       end.to raise_exception("Problem processing review submission.")
       expect(submission.submitted_for_review).to eq(false)
     end
+
+    it "successfully submits app for review if `reject_if_waiting_for_review` option is true" do
```

Comment:
Don't need this if you remove your duplicate method

Reply:
Are you sure we won't need the test cases?

Reply:
I don't think we need this exact test if you remove all the logic out of the `complete` method and make a new method called `reject_if_waiting_for_review ` but we may need a new test case for that method

Reply:
@joshdholtz  I am not very comfortable with removing the logic out of `complete` and creating a new method. I am not sure if I would be able to do that. I am going to keep the code as it is. Feel free to modify my branch and update it.